### PR TITLE
6 validating answers

### PR DESCRIPTION
**Pull request by rstrzalkowski**
* Implemented using external API to check whether words exist in dictionary (polish)
 * Sending answers back to players with check status
 * Created STOMP endpoint to receive votes from players

### CODE REVIEW

File: backend/src/main/java/pl/lodz/p/it/zzpj/service/GameRedisService.java [hunk 1]
```diff
@@ -140,17 +164,62 @@ public void sendAnswers(AnswerDTO answerDTO, UUID gameID)
         }
     }
 
+    public void sendVotes(PlayerVotesDTO playerVotesDTO, UUID gameID)
```
Comment (Msocha19) [inline backend/src/main/java/pl/lodz/p/it/zzpj/service/GameRedisService.java:167]: Add restriction, so that users can only vote once.

Comment (Msocha19) [inline backend/src/main/java/pl/lodz/p/it/zzpj/service/GameRedisService.java:167]: Voting should end after a certain amount of time, I don't think it is needed for everyone to vote.

**Review by Msocha19 — CHANGES_REQUESTED** [review]

File: backend/src/main/java/pl/lodz/p/it/zzpj/service/GameRedisService.java (hunk 1 above)
Comment (rstrzalkowski) [reply backend/src/main/java/pl/lodz/p/it/zzpj/service/GameRedisService.java]: Fixed in updated PR

Comment (rstrzalkowski) [reply backend/src/main/java/pl/lodz/p/it/zzpj/service/GameRedisService.java]: Fixed in updated PR

**Review by Lukasz0104 — APPROVED** [review]

**Pull request merged**